packages/legacy/core/App/localization/en/index.ts

```diff
@@ -297,6 +297,22 @@ const translation = {
     "IssuedBy": "Issued by",
     "Credential": "credential"
   },
+  "PushNotifications": {
+    "EnableNotifiactions": "Enable Notifications to get instant alerts",
+    "BeNotified": "Be notified when you receive:",
+    "ReceiveNotifications": "Receive notifications",
+    "BulletFour": "new messages",
+    "BulletOne": "new credential offers",
+    "BulletThree": "updates to your credentials",
+    "BulletTwo": "new proof requests",
+    "NotificationsOffMessage": "Be notified when you receive new credential offers, proof requests, updates to your credentials, new messages and more.",
+    "NotificationsOffTitle": "Notifications for BC Wallet are turned off",
```

Comment:
I think you meant to put this in BC Wallet, or meant to make it generic to Bifold and then override in BC Wallet.